ravenframework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -169,192 +198,266 @@ def _evaluate(self, datasets, **kwargs):
+    for cnt2 in range(yCount):
+      distanceSum = abs(np.sum(output[cnt2]))
+      sigmaSum = 0
+      for cnt3 in range(zCount):
+        distanceTotal[cnt2][cnt3] = distanceSum
+        sigmaSum += output[cnt2][cnt3]**2
+      for cnt3 in range(zCount):
+        sigma[cnt2][cnt3] = (1/(zCount-np.sum(naNCount[cnt2]))*sigmaSum)**0.5
+    rlz = []
+    for cnt in range(yCount):
       outputDict = {}
-      distanceTotal = np.zeros((yCount,zCount))
-      sigma = np.zeros((yCount,zCount))
-      for metric in self.metrics:
-        name = "{}_{}_{}".format(metric.estimator.name, targ.split("|")[-1], feat.split("|")[-1])
-      output = metric.evaluate((newfeatureData,newtargetData), multiOutput='raw_values')
-      for cnt2 in range(yCount):
-        distanceSum = abs(np.sum(output[cnt2]))
-        sigmaSum = 0
-        for cnt3 in range(zCount):
-          distanceTotal[cnt2][cnt3] = distanceSum
-          sigmaSum += output[cnt2][cnt3]**2
-        for cnt3 in range(zCount):
-          sigma[cnt2][cnt3] = (1/(zCount-np.sum(naNCount[cnt2]))*sigmaSum)**0.5
-      rlz = []
-      for cnt in range(yCount):
-        outputDict = {}
-        outputDict[name] = abs(np.atleast_1d(output[cnt]))
-        outputDict['pivot_parameter'] = timeParameter
-        outputDict['total_distance_'+nameTarg[1]+'_'+nameFeat[1]] = distanceTotal[cnt]
-        outputDict['feature_beta_'+nameTarg[1]+'_'+nameFeat[1]] = featureBeta[cnt]
-        outputDict['target_beta_'+nameTarg[1]+'_'+nameFeat[1]] = targetBeta[cnt]
-        outputDict['feature_omega_'+nameTarg[1]+'_'+nameFeat[1]] = featureOmegaNormScaled[cnt]
-        outputDict['target_omega_'+nameTarg[1]+'_'+nameFeat[1]] = targetOmegaNorm[cnt]
-        outputDict['feature_D_'+nameTarg[1]+'_'+nameFeat[1]] = featureD[cnt]
-        outputDict['target_D_'+nameTarg[1]+'_'+nameFeat[1]] = targetD[cnt]
-        outputDict['process_time_'+nameTarg[1]+'_'+nameFeat[1]] = newfeatureData[1][cnt]
-        outputDict['standard_deviation_'+nameTarg[1]+'_'+nameFeat[1]] = sigma[cnt]
-        rlz.append(outputDict)
-      realizationArray.append(rlz)
+      outputDict[name] = abs(np.atleast_1d(output[cnt]))
+      outputDict['pivot_parameter'] = timeParameter
+      outputDict['total_distance_'+nameTarg[1]+'_'+nameFeat[1]] = distanceTotal[cnt]
+      outputDict['feature_beta_'+nameTarg[1]+'_'+nameFeat[1]] = featureBeta[cnt]
+      outputDict['target_beta_'+nameTarg[1]+'_'+nameFeat[1]] = targetBeta[cnt]
+      outputDict['feature_omega_'+nameTarg[1]+'_'+nameFeat[1]] = featureOmegaNormScaled[cnt]
+      outputDict['target_omega_'+nameTarg[1]+'_'+nameFeat[1]] = targetOmegaNorm[cnt]
+      outputDict['feature_D_'+nameTarg[1]+'_'+nameFeat[1]] = featureD[cnt]
+      outputDict['target_D_'+nameTarg[1]+'_'+nameFeat[1]] = targetD[cnt]
+      outputDict['process_time_'+nameTarg[1]+'_'+nameFeat[1]] = newfeatureData[1][cnt]
+      outputDict['standard_deviation_'+nameTarg[1]+'_'+nameFeat[1]] = sigma[cnt]
+      rlz.append(outputDict)
```

Comment:
please document these outputs in the manual

Reply:
Has been documented as mentioned in previous comment.